doc/deployment/deployment.md

```diff
@@ -63,6 +63,15 @@ This will bump the `minor` version, create a tag `vX.X+1.X` and push it.
 This will bump the `patch` version, create a tag `patch/vX.X.X+1` and push it.
 CircleCI will detect the tag `patch/vX.X.X` and launch the lanes `deploy-ios-staging-hard` & `deploy-android-staging-hard` (see `.circleci/config.yml` file)
 
+#### Patch staging with additionnal commits
+
+Pull master to make sure you have all the commits you need to patch.
+Them you checkout to the actual staging branch which can be found [here][8]
+
+- `git checkout vX.X.X`
```

Comment:
puis 

- list all commits you want to patch to your tag
- run `git cherry-pick <commit-hash>` for all commits __in the order they were merged to `master`__ to avoid conflicts.
- build and deploy the patch by running the following script: `trigger:staging:deploy:patch`